packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-expandable-answer.pcss

```diff
@@ -0,0 +1,36 @@
+@import '../../global/global.pcss';
+@import '../../global/mixins.pcss';
+
+:host {
+  atomic-smart-snippet-answer {
+    @mixin set-font-size var(--atomic-text-lg);
+
+    display: block;
+    overflow: hidden;
+    height: var(--collapsed-size);
+    transition: height ease-in-out 0.25s;
+
+    --gradient-start: var(
+      --atomic-smart-snippet-gradient-start,
+      calc(max(var(--collapsed-size) - (var(--line-height) * 1.5), var(--collapsed-size) * 0.5))
+    );
+    color: var(--atomic-on-background);
+    mask-image: linear-gradient(black, black var(--gradient-start), transparent 100%);
```

Comment:
Unlike the old SmartSnippet, I chose not to render a white gradient on top of the text. Instead, I decided to apply a mask.

The flaw with this solution is that the mask instantly appears/disappears when we press "show less"/"show more". 

An advantage is that, if somebody renders the snippet on top of a background or on top of other components, it should still look "right".